Weather.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+#include "string"
+#include "iostream"
+
+using namespace std;
+
+enum TYPE
+{
+	SUNNY,
```

Comment:
Why are all the letters capitalized? Rename please)

Reply:
Because they should be capitalized, as part of standart enum construction:)